test/sql/aggregate/aggregates/test_string_agg_many_groups.test_slow

```diff
@@ -20,3 +22,14 @@ SELECT 1, STRING_AGG(x, ',') FROM strings GROUP BY 1 ORDER BY 1
 statement error
 SELECT STRING_AGG(k, ','), SUM(CAST(k AS BIGINT)) FROM (SELECT CAST(g AS VARCHAR) FROM strings UNION ALL SELECT CAST(x AS VARCHAR) FROM strings) tbl1(k)
 
+query IT
+SELECT 1, STRING_AGG(x, ',' ORDER BY g DESC) FROM strings GROUP BY 1 ORDER BY 1
+----
+2 values hashing to 3bd0d16b476d2ffe18a77ebe1098b89c
+
+query I
+SELECT STRING_AGG(s, ',' ORDER BY id DESC)
+FROM (SELECT i id, i::VARCHAR s FROM range(1000000) tbl(i)) t
```

Comment:
Scans over the `range` function do not support parallelism (yet?!), so this doesn't test the parallelism correctly right now. A temporary table will need to be created here unfortunately!